src/components/Search/SearchView.jsx

```diff
@@ -14,7 +16,12 @@ export default function SearchView() {
           PerfCompare
         </Typography>
       </Box>
-      <SearchInput />
+      <Grid container>
+        <Grid item xs={2} />
+        <SearchDropdown />
+        <SearchInput />
+        <Grid item xs={2} />
+      </Grid>
```

Comment:
using grid and spacers so that elements will display as expected now that they are in separate components